.editorconfig

```diff
@@ -8,12 +8,20 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.{java,kt}]
+ij_kotlin_imports_layout = *
+
 # Disable wildcard imports
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
 ij_java_class_count_to_use_import_on_demand = 999
-# Enable trailing comma
-ktlint_disabled_rules=trailing-comma-on-call-site,trailing-comma-on-declaration-site
+
+ktlint_code_style = android_studio
+
+# Disable trailing comma
+ij_kotlin_allow_trailing_comma = false
+ij_kotlin_allow_trailing_comma_on_call_site = false
```

Comment:
Again? 😢

Reply:
?

Reply:
Does this disallow *usage* of trailing commas, or disable *forced* usage of trailing commas?

Reply:
I would recommend disabling or enabling it forcibly, instead of using either state.

Reply:
Yes, we want no warning either way.

I understand the urge to force it one way or the other, but it is *really* not worth it.